content/applications/inventory_and_mrp/barcode/operations/barcode_nomenclature.rst

```diff
@@ -1,84 +1,240 @@
+
+Ensure the products scan properly by navigating to the :menuselection:`Barcode app --> Operations`.
+Next, select any operation type, such as :guilabel:`Receipts`, and click the :guilabel:`New` button
+to create a draft stock move. Scan the product weight barcode, such as `2112345015002`, and if the
+intended product appears, the barcode setup is correct.
+
+.. image:: barcode_nomenclature/barcode-scan.png
+   :align: center
+   :alt: Show successfully scanned barcode.
+
+Create rules
+============
+
+.. important::
+   Adding new rules is necessary for |UPC| and |EAN| formats not in Odoo's default list, as barcodes
+   cannot be read successfully if there are unknown fields.
+
+    While new rules can be created, Odoo fields will not auto-populate with information from these
+    rules. `Custom development <https://www.odoo.com/appointment/132>`_ is required for this
+    functionality.
+
+To create a rule, first enable :ref:`developer mode <developer-mode>`. Then, navigate to
+:menuselection:`Inventory app --> Configuration --> Barcode Nomenclatures` menu, and select
+:guilabel:`Default Nomenclature`.
+
+On this page, configure the following optional fields:
+
+- :guilabel:`UPC/EAN Conversion`: determines if a |UPC|/|EAN| barcode should be automatically
+  converted when matching a rule with another encoding. Options include :guilabel:`Always` (the
+  default option), :guilabel:`Never`, :guilabel:`EAN-13 to UPC-A`, and :guilabel:`UPC-A to EAN-13`.
+- :guilabel:`Is GS1 Nomenclature`: ensure this checkbox is **not** ticked, as the
+  :guilabel:`Default Nomenclature` uses |UPC| and |EAN| encoding, *not* GS1 encoding.
+
+.. image:: barcode_nomenclature/rule-config.png
+   :align: center
+   :alt: Default Nomenclature page setting fields.
+
+On the :guilabel:`Default Nomenclature` page, click :guilabel:`Add a line` at the bottom of the
+table, which opens a :guilabel:`Create Rules` pop-up window to create a new rule.
+
+The :guilabel:`Rule Name` field is used internally to identify what the barcode represents.
+
+The :guilabel:`Sequence` field represents the priority of the rule; meaning the smaller the value,
+the higher the rule appears on the table.
+
+The barcode :guilabel:`Type` field represents different classifications of information that can be
+understood by the system (e.g., :guilabel:`Package`, :guilabel:`Lot`, :guilabel:`Location`,
+:guilabel:`Coupon`, etc.).
+
+The :guilabel:`Encoding` field specifies which encoding the barcode uses; this rule **only** applies
+if the barcode uses this specific encoding. The available :guilabel:`Encoding` options are:
+:guilabel:`EAN-13`, :guilabel:`EAN-8`, :guilabel:`UPC-A`, and :guilabel:`GS1-28`.
+
+The :guilabel:`Barcode Pattern` field represents how the sequence of letters or numbers is
+recognized by the system to contain information about the product. Sometimes, when a certain amount
+of digits are required, the number of `.` is shown. `N` represents whole number digits, and `D`
```

Comment:
```suggestion
of digits are required, the figure of `.*` is shown. `N` represents whole number digits, and `D`
```
That's, at least, how it appears during my run-through...

![Screenshot 2024-07-24 080749](https://github.com/user-attachments/assets/0cd09fa2-8457-439d-99a7-d79fcda9153e)

Reply:
I know it's a little bit confusing since I mentioned it while this pop-up is in view, but this section of the text:

> Sometimes, when a certain amount of digits are required, the number of `.` is shown. `N` represents whole number digits, and `D`

Is actually referring to the barcode patterns in the barcode nomenclature list

![image](https://github.com/user-attachments/assets/9aa1b0a0-6ff9-4669-a63d-2520f63cac2e)

If you have suggestions on how to improve the documentation to reflect that, feel free to let me know (: